configure.ac

```diff
@@ -65,7 +65,7 @@ PKG_CHECK_MODULES(GL_TEST, xcomposite gl glib-2.0 epoxy)
 
 PKG_CHECK_MODULES(DBUS_GLIB, dbus-glib-1 >= $DBUS_GLIB_REQUIRED)
 
-PKG_CHECK_MODULES(EGG_SMCLIENT, gtk+-3.0)
+PKG_CHECK_MODULES(LIBEGG, gtk+-3.0)
```

Comment:
It should also include the sm module.

Reply:
Or even better to include the sm and ice modules and remove them from the [Makefile.am](https://github.com/mate-desktop/mate-submodules/blob/799639b54e6e5c2b04a3cd13610ebe54d21b71af/libegg/Makefile.am#L35):
```
$ pkg-config --libs --cflags sm ice
-lSM -lICE 
```
otherwise the library is included twice (-lSM  -lSM):
```
atril $ ./autogen.sh --prefix=/usr && make V=1 &> make.log
atril $ ./autogen.sh --prefix=/usr && make V=1 &> make.log
<cut>
/bin/sh ../../../libtool  --tag=CC   --mode=link gcc -DEGG_SM_CLIENT_BACKEND_XSMP -DG_LOG_DOMAIN=\""EggSMClient"\" -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/harfbuzz -I/usr/include/fribidi -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gio-unix-2.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/at-spi-2.0 -pthread  -Wall -Wmissing-prototypes  -g -O2   -o libegg.la  libegg_la-eggtreemultidnd.lo  libegg_la-eggdesktopfile.lo libegg_la-eggsmclient.lo libegg_la-eggsmclient-xsmp.lo   -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lgthread-2.0 -pthread -lglib-2.0 -lSM  -lSM -lICE -lm 
<cut>
```